dask/tests/test_distributed.py

```diff
@@ -659,3 +659,27 @@ async def test_pack_MaterializedLayer_handles_futures_in_graph_properly(c, s, a,
     packed = hlg.__dask_distributed_pack__(c, ["z"], {})
     unpacked = HighLevelGraph.__dask_distributed_unpack__(packed)
     assert unpacked["deps"] == {"x": {fut.key}, "y": {fut.key}, "z": {"y"}}
+
+
+@gen_cluster(client=True)
+async def test_to_sql_engine_kwargs(c, s, a, b):
+    # https://github.com/dask/dask/issues/8738
+    pd = pytest.importorskip("pandas")
+    dd = pytest.importorskip("dask.dataframe")
+    sql = pytest.importorskip("dask.dataframe.io.sql")
```

Comment:
I'm not sure that this `importorskip` will skip when we want it to as our `sqlalchemy` imports are protected inside, for example, `read_sql_table`

https://github.com/dask/dask/blob/e3b3259419c21d0d412b9d5f12531ebe5ad6967a/dask/dataframe/io/sql.py#L82

Instead we should probably use `importorskip("sqlalchemy")` like is done over in our other sql-related tests 

https://github.com/dask/dask/blob/510bbc380531cbf56a409f1ae68e6fd84a9599e6/dask/dataframe/io/tests/test_sql.py#L14